test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -177,7 +199,7 @@ BOOST_AUTO_TEST_CASE(double_stateVariable_declaration)
 					   "  uint256 variable;\n"
 					   "  uint128 variable;\n"
 					   "}\n";
-	BOOST_CHECK(expectError(text) == Error::Type::DeclarationError);
+	CHECK_ERROR(text, DeclarationError, "");
```

Comment:
Functions will report the wrong failure location, and I think we should check for a message everywhere anyway.